src/components/items.jsx

```diff
@@ -0,0 +1,66 @@
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useParams } from 'react-router-dom';
+import '../styles/Details.css';
+import { useEffect } from 'react'; 
+import { fetchItemInfo } from '../redux/itemsSlice';
+// import { selectedItem } from '../redux/itemsSlice';
```

Comment:
- [ ] Kindly, remove this piece of old commented code in order to improve the repo's cleanliness.  👍